File: digits/digits.py
def digits(arr):
    arr = arr[::-1]
    one = 1
    i = 0
    while one:
        if i <len(arr):
            if arr[i] == 9:
                arr[i] = 0
            else:
                arr[i] +=1
                one = 0
        else:
            arr.append(1)
            one=0
        i += 1
    return arr[::-1]

File: digits/digits.js
function digits(arr){
    arr= arr.reverse();
    let one = 1
    let i = 0
    while (one){
        if (i<arr.length){
            if(arr[i] == 9){
                arr[i] = 0

            }else{
                arr[i]+=1
                one =0
            }
        }
        else{
            arr.push(1)
            one = 0
        }
        i++;
    }
    return arr.reverse()
    
}